File: Week3/Inst/app/src/main/java/com/example/inst/MainActivity.java
package com.example.inst;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.ImageButton;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    ImageButton ibLike;
    ImageButton ibSave;
    ImageButton ibMore;
    boolean liked = false;
    boolean saved = false;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        ibLike = findViewById(R.id.ib5);
        ibLike.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                if (!liked){
                    ibLike.setImageResource(R.drawable.liked);
                    liked = true;
                } else {
                    ibLike.setImageResource(R.drawable.like);
                    liked = false;
                }
            }
        });

        ibSave = findViewById(R.id.ib8);
        ibSave.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                if (!saved){
                    ibSave.setImageResource(R.drawable.saved);
                    saved = true;
                } else {
                    ibSave.setImageResource(R.drawable.save);
                    saved = false;
                }
            }
        });

        ibMore = findViewById(R.id.ib4);
        ibMore.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Toast.makeText(
                        MainActivity.this,
                        "You have opened Menu",
                        Toast.LENGTH_SHORT
                ).show();
            }
        });

    }
}

File: IviApp/app/src/main/java/com/example/iviapp/MainActivity.kt
package com.example.iviapp

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val button: Button = findViewById(R.id.button)
        button.setOnClickListener(){
            val intent = Intent(this@MainActivity, SecondActivity::class.java)
            intent.putExtra("news", 1)
            intent.putExtra("index", 1)
            startActivity(intent)
        }
    }
}
